docling_convert_components.py

```diff
@@ -272,6 +276,10 @@ def docling_convert(
     else:
         pipeline_options = PdfPipelineOptions()
         pipeline_options.artifacts_path = artifacts_path_p
+        pipeline_options.do_code_enrichment = enrich_code
+        pipeline_options.do_formula_enrichment = enrich_formula
+        pipeline_options.do_picture_classification = enrich_picture_classes
```

Comment:
In the example here (https://docling-project.github.io/docling/usage/enrichments/#picture-classification), `pipeline_options.generate_picture_images = True` is set. Do you know if that needs to be set for `do_picture_classification` to take effect?